pkg/graveler/ref/manager_test.go

```diff
@@ -1111,3 +1111,49 @@ func TestManager_IsTokenExpired(t *testing.T) {
 		t.Fatalf("expected token expired")
 	}
 }
+
+func TestManager_DeleteExpiredAddressTokens(t *testing.T) {
+	r, _ := testRefManager(t)
+	ctx := context.Background()
+	repository, err := r.CreateRepository(ctx, "repo1", graveler.Repository{
+		StorageNamespace: "s3://",
+		CreationDate:     time.Now(),
+		DefaultBranchID:  "main",
+	})
+	testutil.Must(t, err)
+
+	a := xid.NewWithTime(time.Now()).String()
+	b := xid.NewWithTime(time.Now().Add(-10 * time.Hour)).String() // expired
+	c := xid.NewWithTime(time.Now().Add(-7 * time.Hour)).String()  // expired
+	prefix := "data/skdgjslk/"
+
+	tokens := []string{prefix + a, prefix + b, prefix + c}
```

Comment:
Can we have different prefixes - similar to what we have in real time (reverse order)

Reply:
done